packages/layers/stage3/build.yaml

```diff
@@ -0,0 +1,14 @@
+image: "quay.io/mocaccino/extra:latest"
+
+package_dir: /gentoo
+
+prelude:
+- luet upgrade && luet install -y utils/tar utils/curl utils/xz
+- mkdir /gentoo
+
+steps:
+- curl -L http://distfiles.gentoo.org/releases/amd64/autobuilds/current-stage3-amd64/stage3-amd64-systemd-20210512T214503Z.tar.xz | tar Jxp --xattrs --numeric-owner -C /gentoo
```

Comment:
```suggestion
- curl -L http://distfiles.gentoo.org/releases/amd64/autobuilds/current-stage3-amd64/stage3-amd64-systemd-${STAGE_VERSION}.tar.xz | tar Jxp --xattrs --numeric-owner -C /gentoo
```

Reply:
Also, if you want to make it `multi-arch`, use a `{{.Values.arch}}` in place and define a default one inside `definition.yaml`